LayoutTests/fast/events/page-visibility-onvisibilitychange-in-iframe.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<body onload="startTest()">
+<iframe id="iframeTest" src="resources/page-visibility-iframe-send-onvisibility-change-back.html"></iframe>
```

Comment:
This is not a cross-site iframe.  We need something like LayoutTests/http/tests/site-isolation/basic-iframe.html that uses localhost for the iframe and implicitly 127.0.0.1 for the main frame.

Reply:
Fixed